modules/cms-magento/components/SfxMagentoCmsBlock.vue

```diff
@@ -16,5 +20,15 @@ const props = defineProps({
   },
 })
 
-const { data } = await useAsyncData(`getCMSBlock_${props?.identifier}`, () => getCmsBlock(props.identifier))
+let cmsData: Awaited<ReturnType<typeof getCmsBlock>> | null = null
+
+try {
+  const { data, error } = await useAsyncData(`getCMSBlock_${props?.identifier}`, () => getCmsBlock(props.identifier))
+  if (error.value) {
+    showErrorNotification(error.value)
```

Comment:
I think there is a more elegant solution to error handling here. Lets discuss it later.